src/locale/en.yml

```diff
@@ -156,7 +156,7 @@ feat:
         remove: Remove from assignment
       add:
         alreadyAdded: Already added
-        placeholder: Add caller
+        placeholder: Start typing to search or add a new caller
```

Comment:
Instead of changing this, you should change the message in `messageIds.ts` and remove the translation from `en.yml` and all other YAML files.

Reply:
-> b4fb79d